library/src/test/java/com/datatorrent/lib/helper/OperatorContextTestHelper.java

```diff
@@ -44,22 +42,26 @@ protected DateFormat initialValue()
 
   };
 
-  public static class TestIdOperatorContext extends TestContext implements OperatorContext
+  public abstract static class TestIdOperatorContext extends TestContext implements OperatorContext
```

Comment:
I think the class should remain public to be able to extend it. I see the need to possibly add builder style setters in the future to make it easier to customize the context for test cases and hence I prefer to keep the class exposed. I don't have a strong opinion on where the factory methods reside (one level up would make it easier to use though), it wasn't the focus of the PR. If you think it is important enough, I can make that change, I would also rename the class to `TestOperatorContext` then.